yt/python/yt/wrapper/tests/test_docker_respawn.py

```diff
@@ -0,0 +1,190 @@
+import tempfile
+
+from .conftest import authors
+
+from yt.wrapper.py_wrapper import DockerRespawner, respawn_in_docker
+
+
+@authors("thenno")
+def test_docker_respawner(monkeypatch):
+    monkeypatch.setenv("SOME_KEY", "SOME_VALUE")  # should be skipped
+    monkeypatch.setenv("YT_SOME_KEY", "YT_SOME_VALUE")
+    monkeypatch.setenv("ANOTHER_KEY", "ANOTHER_VALUE")  # should be skipped
+    respawner = DockerRespawner(
+        image="some_image",
+        target_platform="arm64",
+        docker="docker_test",
+        python="/custom/docker/python/path",
+        env=None,
+        main_scipt_path="/home/user2/yt/main.py",
+        cwd="/root",
+        homedir="/home/user",
+        python_lib_paths=[
+            "/usr/lib/python4.2/site-packages",
+            "/home/user/.venv/",  # in homedir -> not in mounts
+        ],
+    )
+    assert respawner.make_command() == [
+        "docker_test",
+        "run",
+        "--platform", "arm64",
+        "-it",
+        "--rm",
+        "-e", "CWD=/root",
+        "-e", "PYTHONPATH=/usr/lib/python4.2/site-packages:/home/user/.venv/",
+        "-e", "YT_RESPAWNED_IN_CONTAINER=1",
+        "-e", "YT_SOME_KEY=YT_SOME_VALUE",
+        "-e", "YT_BASE_LAYER=some_image",
+        "-v", "/home/user:/home/user",  # user's homedir
+        "-v", "/home/user2/yt:/home/user2/yt",  # main script's dir
+        "-v", "/root:/root",  # current cwd
```

Comment:
Let here (and in other places in this file) fix the comments:
* Write it on the previous line.
* Write it as `# Specifying user's homedir.`